app/controllers/admin/collections_controller.rb

```diff
@@ -217,4 +217,40 @@ def destroy
       redirect_to admin_collection_path(@source_collection)
     end
   end
+
+  # POST /collections/1/batch_ingest
+  def batch_ingest
+    if @collection.dropbox.manifest_processable?
+      # Spawn off background job
+      AvalonJobs.collection_batch_ingest(@collection.id)
+      flash[:notice] = t('batch_ingest.initiated', name: @collection.name)
+    else
+      flash[:error] =
+        if @collection.dropbox.manifest_error?
```

Comment:
If there just for testing, then I would push back and say remove them (YAGNI). Your just using them for simple expects on the status value regardless:
```
expect(collection.dropbox.manifest_processable?).to be_truth
```
If you do the refactor on https://github.com/ualbertalib/avalon/pull/150/files#r120497550

Then you can just use the status return value instead (probably a symbol?)
```
expect(collection.dropbox.manifest_status).to eq(:processable)
```
If you dont like the string interpolation of the i18n value using the status value, then have your `locate_with_status` return the actual status message instead of a status symbol (no point doing multiple if statements multiple times)